.circleci/config.yml

```diff
@@ -154,7 +161,7 @@ jobs:
             touch -m -d '1 Jan 2021 12:00' emsdk/emsdk/.emscripten
 
             ccache -z
-            PYODIDE_PACKAGES='<< parameters.packages >>' make dist/pyodide-lock.json
+            ENABLE_PREBUILT_PACKAGES='<< parameters.enable_prebuilt_packages >>' PYODIDE_PACKAGES='<< parameters.packages >>' make dist/pyodide-lock.json
```

Comment:
I don't see an implementation for `ENABLE_PREBUILT_PACKAGES`...

Reply:
It is used in the [packages/Makefile](https://github.com/pyodide/pyodide/blob/95dd5285866ec60007814cae7032ff31976dd4c2/packages/Makefile)

Reply:
Ah, right. Could you split the PR into a first one that makes the CI and Makefile changes and a second that only removes the recipes? Taking out the recipes adds a lot of noise to the diff.

Reply:
Sure, let me try to split the PRs